app/Services/ocr_services.py

```diff
@@ -18,11 +18,10 @@ def __init__(self):
     def _image_preprocess(img: Image.Image) -> Image.Image:
         if img.mode != 'RGB':
             img = img.convert('RGB')
-        if img.size[0] > 1024 or img.size[1] > 1024:
-            img.thumbnail((1024, 1024), Image.Resampling.LANCZOS)
-        new_img = Image.new('RGB', (1024, 1024), (0, 0, 0))
-        new_img.paste(img, ((1024 - img.size[0]) // 2, (1024 - img.size[1]) // 2))
-        return new_img
+        # Limit maximum size to 960*960
+        if img.size[0] > 960 or img.size[1] > 960:
+            img.thumbnail((960, 960), Image.Resampling.LANCZOS)
+        return img
```

Comment:
Since the optimized `easypaddleocr` doesn't need the scaling process, while other ocr services do, **it's better to rewrite `_image_preprocess` only in the `EasyPaddleOCRService` class instead of modifying `_image_preprocess` in the base class**